feat: gnomprona monsters and npcs

murilo09 opened the pull request:
# Description

Gnomprona Monsters (https://github.com/opentibiabr/otservbr-global/pull/786) and Npcs

# **Feel free to contribute!**


# Monsters

## Sparkling Pools
![190534823-fe23e875-fa36-428a-b4f2-9b3dfd633b27](https://user-images.githubusercontent.com/78226931/234319399-25ff6c1c-2f63-4874-91e8-e927e487feac.png)

- Emerald Tortoise
- Gore Horn
- Gorerilla
- Hulking Prehemoth
- Sabretooth

## Monster Graveyard
![190533446-238b1a12-4622-4e8b-be4c-f782667b1280](https://user-images.githubusercontent.com/78226931/234319646-31a15fa7-8f94-4d78-a84d-f988b5ff2ab2.png)

- Nighthunter
- Stalking Stalk
- Sulphider
- Sulphur Spouter
- Undertaker

## Crystal Enigma
![190507196-7cfff2d1-f61b-4e7b-827c-c33ccd135a78](https://user-images.githubusercontent.com/78226931/234319829-6903af33-40e6-4dff-8b38-6a7830dc518c.png)

- Headpecker
- Mantosaurus
- Mercurial Menace
- Noxious Ripptor
- Shrieking Cry-stal

## Type of change

  - [X] New feature (non-breaking change which adds functionality)

murilo09 committed:
monsters and npcs

murilo09 committed:
some adjustments

QuebradaZN reviewed (changes requested):
/npc/gnominer.lua:57: ')' expected (to close '(' at line 53) near

murilo09 committed:
fixes

QuebradaZN reviewed (changes requested):
{id = 3007, chance = 1430}, --crystal ring

QuebradaZN commented on data-otservbr-global/monster/elementals/sulphur_spouter.lua:
```diff
@@ -0,0 +1,124 @@
+	{name = "Crystal Coin", chance = 22310, minCount = 1, maxCount = 2},
+	{name = "Ultimate Mana Potion", chance = 12890, minCount = 1, maxCount = 2},
+	{name = "Yellow Gem", chance = 2820},
+	{name = "Giant Shimmering Pearl (Green)", chance = 2450},
```
```suggestion
	{id = 282, chance = 2450}, --Giant Shimmering Pearl (Green)
```

QuebradaZN commented on data-otservbr-global/monster/elementals/sulphur_spouter.lua:
```diff
@@ -0,0 +1,124 @@
+	{name = "Knight Legs", chance = 2360},
+	{name = "Red Gem", chance = 2190},
+	{name = "Warrior's Shield", chance = 1760},
+	{name = "Ring of Red Plasma", chance = 1110},
```
```suggestion
	{id = 23533, chance = 1110}, --ring of red plasma
```

QuebradaZN commented on data-otservbr-global/monster/elementals/sulphur_spouter.lua:
```diff
@@ -0,0 +1,124 @@
+	{name = "Giant Shimmering Pearl (Green)", chance = 2450},
+	{name = "Slightly Rusted Legs", chance = 2420},
+	{name = "Knight Legs", chance = 2360},
+	{name = "Red Gem", chance = 2190},
```
```suggestion
	{id = 3039, chance = 2190}, --Red Gem
```

QuebradaZN commented on data-otservbr-global/monster/mammals/gore_horn.lua:
```diff
@@ -0,0 +1,120 @@
+}
+
+monster.loot = {
+	{name = "Gore Horn (Item)", chance = 36040},
```
```suggestion
	{name = "Gore Horn", chance = 36040}, --Gore Horn Item
```

QuebradaZN commented on data-otservbr-global/monster/mammals/gore_horn.lua:
```diff
@@ -0,0 +1,120 @@
+	{name = "Gore Horn (Item)", chance = 36040},
+	{name = "Crystal Coin", chance = 30050},
+	{name = "Big Bone", chance = 5270},
+	{name = "Dwarven Ring", chance = 3590},
```
```suggestion
	{id = 3097, chance = 3590}, --Dwarven Ring
```

QuebradaZN commented on data-otservbr-global/monster/mammals/parder.lua:
```diff
@@ -0,0 +1,120 @@
+	{name = "Red Crystal Fragment", chance = 14030},
+	{name = "Strong Health Potion", chance = 7560, minCount = 1, maxCount = 3},
+	{name = "Barbarian Axe", chance = 4540},
+	{name = "Crimson Sword", chance = 1510},
```
```suggestion
	{id = 7385, chance = 1510}, --Crimson Sword
```

QuebradaZN commented on data-otservbr-global/monster/reptiles/emerald_tortoise.lua:
```diff
@@ -0,0 +1,126 @@
+	{name = "Green Crystal Shard", chance = 2900},
+	{name = "Green Gem", chance = 2870},
+	{name = "Orichalcum Pearl", chance = 2640, minCount = 1, maxCount = 2},
+	{name = "Giant Shimmering Pearl (Green)", chance = 2420},
```
```suggestion
	{id = 282, chance = 2420}, --Giant Shimmering Pearl (Green)
```

QuebradaZN commented on data-otservbr-global/monster/reptiles/emerald_tortoise.lua:
```diff
@@ -0,0 +1,126 @@
+	{name = "Green Gem", chance = 2870},
+	{name = "Orichalcum Pearl", chance = 2640, minCount = 1, maxCount = 2},
+	{name = "Giant Shimmering Pearl (Green)", chance = 2420},
+	{name = "Red Gem", chance = 2420},
```
```suggestion
	{id = 3039, chance = 2420}, --Red Gem
```

QuebradaZN commented on data-otservbr-global/monster/reptiles/mantosaurus.lua:
```diff
@@ -0,0 +1,123 @@
+	{name = "Cyan Crystal Fragment", chance = 3010},
+	{name = "Green Crystal Shard", chance = 3010},
+	{name = "Coral Brooch", chance = 2420},
+	{name = "Crystal Ring", chance = 1430},
```
```suggestion
	{id = 3007, chance = 1430}, --Crystal Ring
```

murilo09 committed:
Change item name to id

Co-Authored-By: Douglas dos Santos <47014952+QuebradaZN@users.noreply.github.com>

LeoBorges18 commented on data-otservbr-global/scripts/quests/primal_ordeal_quest/the_primal_manace_lever.lua:
```diff
@@ -0,0 +1,93 @@
+	end
+end
+
+PrimalMenaceLever:position({33547, 32752, 14})
```
PrimalMenaceLever:position({x = 33547, y = 32752, z = 14})

jilikuo commented on data-otservbr-global/monster/elementals/sulphur_spouter.lua:
mindamage do melee não deveria ser -300 ou é intencional ser 300?

murilo09 committed:
Fix Primal Menace Lever

Co-Authored-By: LeoBorges <125532357+leoborges18@users.noreply.github.com>

murilo09 committed:
Fix Sulphur Spouter melee damage

Co-Authored-By: jilikuo <54220633+jilikuo@users.noreply.github.com>

murilo09 committed:
Levers to new standard

lennonpcwb reviewed (commented):
![image](https://user-images.githubusercontent.com/113062642/235736023-09c58113-52fa-4a64-a13d-67ffff0b370b.png)

Não abre a bag de reward do The Primal Menace após a morte, dizendo que já foi recebido!

victortp commented:
> ![image](https://user-images.githubusercontent.com/113062642/235736023-09c58113-52fa-4a64-a13d-67ffff0b370b.png)
> 
> Não abre a bag de reward do The Primal Menace após a morte, dizendo que já foi recebido!

É porque o primal menace não dropa loot

murilo09 committed:
Remove reward from The Primal Menace

libergod commented on data-otservbr-global/monster/birds/headpecker.lua:
I've received an error because of the loot "Carrot" which has more than 1 itemid possible in Items.xml. What i've done was changing from : 
`{name = "Carrot", chance = 4950},`

to
`{id = 3595, chance = 4950, minCount = 1, maxCount = 3},`

carlospess0a committed:
Removing Carrot due conflit, adding id

murilo09 committed:
Update headpecker.lua

dudantas committed:
add trailing newlines to end of files

This addition is a recommended practice to ensure consistency and prevent formatting issues, linter warnings, and merge conflicts. Including trailing newlines improves code readability and enables smooth integration.

dudantas committed:
Merge remote-tracking branch 'upstream/main' into gnomprona

dudantas committed:
fix: non-unique item warning
